lib/views/components/content/publication_header/user_profile_pop_up.dart

```diff
@@ -0,0 +1,42 @@
+import "package:flutter/material.dart";
+import "package:proxima/models/ui/user_avatar_details.dart";
+import "package:proxima/views/components/content/user_avatar/user_avatar.dart";
+import "package:proxima/views/helpers/key_value_list_builder.dart";
+
+class UserProfilePopUp extends StatelessWidget {
```

Comment:
Could you add some documentation for this class please?

Reply:
Ah yes indeed, this is a good idea. Fixed in 71237ce24f39bbebef736f75831ca6ac92c11db7.